index.html

```diff
@@ -5929,8 +5932,8 @@ <h3>Key Management</h3>
         <h3>Content Integrity Protection</h3>
 
         <p>
-[=Verifiable credentials=] often contain URLs to data that resides outside of
-the [=verifiable credential=] itself. Linked content that exists outside a
+[=Verifiable credentials=] often contain URLs to data that resides outside
+the [=verifiable credential=]. Linked content that exists outside a
 [=verifiable credential=], such as images, JSON-LD extension contexts,
 JSON Schemas, and other machine-readable data, are not protected by default
```

Comment:
hmm removing the middle clause "JSON Schemas is not protected" does not make sense

since data is plural I believe the are works here

Reply:
The simplified phrase that should agree is `Linked content ... is not protected`.

Probably this would be clearer if the comma separators around this list of examples were changed to emdashes, as I did elsewhere... i.e.,
```
Linked content that exists outside a
[=verifiable credential=] &mdash; such as images, JSON-LD extension contexts,
JSON Schemas, and other machine-readable data &mdash; is not protected by default
```

Reply:
agree with that. would you mind amending the suggestion?

Reply:
moved to https://github.com/w3c/vc-data-model/pull/1563/files#r1764050506